c++/include/orc/orc-config.hh.in

```diff
@@ -25,6 +25,7 @@
   #include <stdint.h>
 #endif
 
+// Following MACROS should be keeped for backward compatibility.
```

Comment:
Just a question. Do we have any deprecation process in Apache ORC C++, @wgtmac , @stiga-huang ?
I'm wondering when we can clean up these unused macros in the future.

Reply:
> Just a question. Do we have any deprecation process in Apache ORC C++, @wgtmac , @stiga-huang ? I'm wondering when we can clean up these unused macros in the future.

IMHO, these kinds of breaking change (e.g. remove deprecated stuff) can be included in a major release. C++14 adds the support of `[[deprecated]]` and we can leverage it to deal with the change of public API. But it is tricky to deal with macros (e.g. add a new macro at the compile time to enable the deprecated macros).

In short, we should keep backward compatibility until the next major release (e.g. v2.0.0) and let the downstream know anything is going to be deprecated.

Reply:
Got it. Thank you. I agree with you. We can do that later separately if needed.